Content.Shared/ReagentOnItem/Components/SpaceGlueOnItemComponent.cs

```diff
@@ -0,0 +1,13 @@
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom;
+
+namespace Content.Shared.ReagentOnItem;
+
+[RegisterComponent]
+public sealed partial class SpaceGlueOnItemComponent : ReagentOnItemComponent
+{
+    [DataField("timeOfNextCheck", customTypeSerializer: typeof(TimeOffsetSerializer)), ViewVariables(VVAccess.ReadWrite)]
```

Comment:
And tag the field with this attribute.
```suggestion
    [DataField, AutoPausedField]
```
(And get rid of that other stuff; it's outdated)

Reply:
Thanks! I did that and also removed all the other outdated stuff in the other components 👏